problems/removeNumbersAtOddIndices.js

```diff
@@ -1,9 +1,21 @@
 /** Remove all numbers at odd indices from an array
-* @param {number[]} arr - The input array
-* @returns {number[]} - An array removing all elements initially appearing at an odd index
-*/
-function removeNumbersAtOddIndices() {
-
+ * @param {number[]} arr - The input array
+ * @returns {number[]} - An array removing all elements initially appearing at an odd index
+ */
+function removeNumbersAtOddIndices(arr) {
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
This works! Another way to tackle this is with a filter that checks for i % 2 === 0